newrelic/resource_newrelic_one_dashboard.go

```diff
@@ -444,6 +453,22 @@ func dashboardWidgetNullValuesSchemaElem() *schema.Resource {
 		},
 	}
 }
+func dashboardWidgetInitialSortingSchemaElem() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"direction": {
+				Type:        schema.TypeString,
+				Required:    true,
```

Comment:
should `direction` be a required field? 🤔 

(It technically should be, but I think you might want to see what happens if we do not send the `direction` with `initialSorting` and only send the `name` - if it defaults to ascending or descending, we might then want to add it as a default value here, after we have a word with the dashboards team about this if necessary)

Reply:
@pranav-new-relic tried sending only name without direction. It doesn't have a default value, and in UI it would show 'Select Value' if we don't specify 'direction' in NerdGraph request. Will confirm with dashboard team as well.